app/grants/views.py

```diff
@@ -1593,6 +1593,9 @@ def grant_edit(request, grant_id):
         if not description_rich:
             description_rich = description
 
+        is_clr_eligible = json.loads(request.POST.get('is_clr_eligible', 'true'))
+        grant.is_clr_eligible = description_rich = is_clr_eligible
```

Comment:
Why are we assigning `description_rich = is_clr_eligible`, this will overwrite  `description_rich`?

Reply:
Ugh issues with copy pasting

Reply:
Awesome! 😅